KeystreamGenerator.rb

```diff
@@ -0,0 +1,138 @@
+class KeystreamGenerator
+
+  attr_reader :keystream_length, :deck
+
+  def initialize(keystream_length)
+    @keystream_length = keystream_length
+  end
+
+  def generate_keystream
+    @deck = build_deck
+    generate_letters
+  end
+
+  private
+
+  NUMBER_CONVERTER = Hash[(1..26).zip ('A'..'Z')]
```

Comment:
moved the constant up. 
will come back to creating a separate class for the Value Object once I've worked through the rest of the comments/suggestions. thanks.